test/Libraries/RevitNodesTests/Elements/ElementTests.cs

```diff
@@ -267,13 +267,19 @@ public void ElementCurveReferences_ExtractsExpectedReferences()
         [TestModel(@".\Element\hostedElements.rvt")]
         public void CanSuccessfullyGetHostedElements()
         {
-            #region Arrange
-            // Select the wall element in revit by it Id
+            // Arrange - Select the wall element in revit by it Id
             var elem = ElementSelector.ByElementId(261723, true);
-            #endregion
-
-            #region Act
-            // Invoke GetHostedElements with all possible cobinations
+            // Model element names
+            string windowFamName = "600 x 3100";
+            string curtainWallFamName = "Curtain Wall";
+            string wallOpeningFamName = "Rectangular Straight Wall Opening";
+            // Assert lists
+            var nameCombo1 = new[] { windowFamName, windowFamName, windowFamName };
```

Comment:
instead of `nameCombo1`, use a descriptive name, like `famNamesWithInserts` ? see the lines below in `Act` for what the names should match up to